src/Installer/Facade/PsAccounts.php

```diff
@@ -41,7 +41,7 @@ public function __construct(Installer $installer)
      */
     public function getService($serviceName)
     {
-        if ($this->installer->isModuleInstalled()) {
+        if ($this->installer->isModuleInstalled() && \Module::getInstanceByName($this->installer->getModuleName()) instanceof \Ps_accounts) {
```

Comment:
Isn't this check redundant or shouldn't we enforce this check elsewhere inside installer class (ie: isModuleInstalled) ?

Reply:
You are right. This is redundant.